snap/remote/remote.go

```diff
@@ -47,4 +46,8 @@ type Snap struct {
 	Title           string             `json:"title"`
 	Type            snap.Type          `json:"content,omitempty"`
 	Version         string             `json:"version"`
+
+	// FIXME: the store should return "developer" to us instead of
+	//        origin
+	Developer string `json:"origin" yaml:"origin"`
```

Comment:
We currently use this "developer" yaml in the snap/removed/removed.go code. As soon as this code dies we don't need the yaml anymore. As soon as the store provides as with "developer" instead of "origin" it will also work naturally (except for upgrade concerns that we might ignore because we are still on rolling).